### PR TITLE
[MAINT] Shellcheck

### LINKED ISSUE #108: Correct Shellcheck Errors
### Summary

There are a litany of shellcheck warnings in the files.

### Actual behavior

The warnings are a first pass at determining acceptable code, but we do not adequately test for them

### Expected behavior

Have the code be "clean" as a first pass and then make sure a test works to keep the code "clean"

### Script/Workflow details

Please put URL to code or code here (if not too long).

### PR DESCRIPTION
**Pull request by jdkent**
Fixes #108

Changes proposed in this pull request
- Going through each file and correct shellcheck warnings
- modify travisci test at the end to automate keeping the code "clean"

This may take a while as I want to run the tests after each file is cleaned to make sure the functionality remains.

### CODE REVIEW

**Comment by jdkent** [comment]
This is ready for review, once all the tests pass and we merge this in, we can have a working suite of tests for both code quality and function.

This is the final item I wanted to work on to make sure going forward this code base does not experience much dilapidation and code rot.

File: PlotPow.sh [hunk 1]
```diff
@@ -132,13 +133,13 @@ fi
...
 if [ "$HighPass" = "1" ] ; then
-    HPsigma=$(echo 100/2/$TR | bc -l)
+    HPsigma=$(echo 100/2/"$TR" | bc -l)
     fslmaths "$Img" -bptf "$HPsigma" -1 $Tmp-HPf
     Img=$Tmp-HPf
 fi
 
 # Center and possibly variance normalize 
-if [ "$VolNorm" = "1" ] ; then
+if [ "$VarNorm" = "1" ] ; then
```
Comment (mwvoss) [inline PlotPow.sh]: how will this be 1 based on setting above?  just marking to ensure changing the variable name was intentional.

Comment (jdkent) [reply PlotPow.sh]: Good catch. Yes, this was intentional, `VolNorm` was not defined anywhere in the script and `VarNorm` [was defined](https://github.com/HBClab/RestingState/pull/109/files#diff-b778ce229ab74c20181d26230cbb3563L70), but never used, so I assumed that was a typo given the behavior of the code below the if appears to be demeaning and dividing the image by the standard deviation.

**Comment by jdkent** [comment]
Alright, I'll merge this in and bump the version to 0.3.3, this way any weird errors that occur moving forward, we will be able to isolate whether it's due to this pull request.

**Pull request merged**